ajo/.env

```diff
@@ -6,3 +6,4 @@ demoSurfaceName="#sample-json-content"
 demoSurfaceUri="web://localhost/"
 hostnames=localhost,aepdemo.com
 AEP_EDGE_DOMAIN="edge.adobedc.net"
+DATASTREAM_ID="8cefc5ca-1c2a-479f-88f2-3d42cc302514"
```

Comment:
is it possible to use the `EDGE_CONFIG_ID_WITH_ANALYTICS` datastream instead?  I think it will be confusing to have multiple edge configs in this file.

Reply:
yes, that works